meta-intel-extras/recipes-core/base-files/base-files/intel-corei7-64/fstab

```diff
@@ -1,3 +1,4 @@
 /dev/root            /                    auto       defaults              1  1
 /dev/sda1	/boot	vfat	defaults	0	0
```

Comment:
The /dev/sda1 and /dev/sda4 are replaced by PARTUUID's so generate PARTUUID for sda5 also.